src/System Application/App/Email/src/Email/EmailImpl.Codeunit.al

```diff
@@ -36,6 +36,8 @@ codeunit 8900 "Email Impl"
         EmailViewPolicyUsedTxt: Label 'Email View Policy is used', Locked = true;
         EmailViewPolicyDefaultTxt: Label 'Falling back to default email view policy: %1', Locked = true;
         EmailModifiedByEventTxt: Label 'Email has been modified by event', Locked = true;
+        AdminViewPolicyInEffectNotificationIdTok: Label '0ee5d5db-5763-4acf-9808-10905a8997d5', Locked = true;
+        AdminViewPolicyInEffectNotificationMsg: Label 'Your email view policy limits the emails visible. Please update your view policy to see all emails.';
```

Comment:
I'm not sure we should say "Please update your view policy", that sounds like something is setup incorrectly and we are asking them to fix it. Maybe just "You can update your view policy to see all emails."